src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Agreement/Agreements.cshtml

```diff
@@ -39,6 +39,8 @@
                 }
             </tbody>
         </table>
+        <p>You can download this data as a <a href="@Url.Action("DownloadCsv", "Agreement")" aria-label="">CSV</a> 
```

Comment:
We should probably slip in a couple of aria-labels giving the full link purpose, eg. "Download agreement data in CSV format".

Reply:
good idea.